Makefile

```diff
@@ -1,5 +1,10 @@
 DUCKDB_REPO=https://github.com/duckdb/duckdb.git
-DUCKDB_REF=5f5512b827df6397afd31daedb4bbdee76520019
+
+export DUCKDB_REF ?= 5f5512b827df6397afd31daedb4bbdee76520019
```

Comment:
i am not sur ethis needs to be exported `DUCKDB_REF` that is.

I have a preference towards using `awk` here rather than exporting and using `envsubst`. Seems odd to change env just to do a templating step